pkg/dwarf/line/state_machine_test.go

```diff
@@ -44,6 +44,10 @@ func TestGrafana(t *testing.T) {
 		t.Skip("filepath.Join ruins this test on windows")
 	}
 
+	if ptrSizeByRuntimeArch() != 8 {
```

Comment:
Remove this `if` and then in this test instead of calling `ptrSizeByRuntimeArch` hardcode 8

Reply:
Oh, I'm sorry for my understanding. I get it.

Reply:
Done